.github/workflows/deploy.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           git config --global user.email 'github-actions@github.com'
 
       - name: Install dependencies
-        run: npm ci
+        run: npm i
```

Comment:
could you please elaborate on this change? as i know `npm ci` and `npm i` do the same thing except that `npm i` also updates `package-lock.json`

Reply:
upon reflection you appear to be correct in using `npm ci` in the CI pipeline according to https://stackoverflow.com/a/53325242/3208553, so i agree that we should revert back to `npm i`

Reply:
```suggestion
        run: npm ci
```